.github/workflows/column2Vec_test.yml

```diff
@@ -0,0 +1,31 @@
+name: "Column2Vec tests"
+
+on:
+  schedule:
+    - cron: '0 10 * * 6'
+  workflow_dispatch:
+
+
+jobs:
+  python-tests:
+    name: Run Python Tests
```

Comment:
Both workflows should not be named Python Tests. Rename this one

Reply:
Fixed